stories/primer/clipboard_copy_stories.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class Primer::ClipboardCopyStories < ViewComponent::Storybook::Stories
+  layout "storybook_preview"
+
+  story(:clipboard_copy_simple) do
+    controls do
+      text(:id, "id")
+      text(:value, "Text to copy")
+      text(:classes, "btn btn-sm")
```

Comment:
```suggestion
      text(:value, "Text to copy")
```

For the sake of these examples, I don't think it's necessary to provide controls for `id` or `classes`.